sdk/servicebus/service-bus/package.json

```diff
@@ -84,10 +97,11 @@
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
-    "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build-browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build-node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build-samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
+    "build-test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
```

Comment:
I would break these out as:

```
"build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
"build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
"build:test": "npm run build:test:node && npm run build:test:browser"
```